ada_feeding_msgs/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   "srv/GetSettings.srv"
   "srv/SetSettings.srv"
   "srv/ToggleFaceDetection.srv"
+  "srv/GetPoseStamped.srv"
```

Comment:
This does not exist anymore, so compilation of `ada_Feeding_msgs` will always fail. Remove this.